src/io/flutter/ProjectOpenActivity.java

```diff
@@ -81,15 +81,15 @@ public void runActivity(@NotNull Project project) {
     @NotNull private final PubRoot myRoot;
 
     public PackagesOutOfDateNotification(@NotNull Project project, @NotNull PubRoot root) {
-      super("Flutter Packages", FlutterIcons.Flutter, "Flutter packages get.",
+      super("Flutter Packages", FlutterIcons.Flutter, "Flutter pub get.",
             null, "The pubspec.yaml file has been modified since " +
-                  "the last time 'flutter packages get' was run.",
+                  "the last time 'flutter pub get' was run.",
```

Comment:
I wonder if we're getting to the point of needing to have all user-visible strings in the bundle.

Reply:
That's likely a useful change to make.

I wonder what tools JetBrains has for this? Are there lints we can enable? Is there a way to know how many non-localized strings we currently have?